docs/t-sql/statements/statements.md

```diff
@@ -39,13 +39,16 @@ Data Definition Language (DDL) statements defines data structures. Use these sta
 - RENAME
 - UPDATE STATISTICS
 
+There are other language elements that affect data structures as well.

```

Comment:
Hi - I do not understand the purpose behind the addition of this sentence. Can you please elaborate or remove? 
Thanks,
Carl

Reply:
I didn't have time to review the entirety of BOL to be sure nothing else applies, so I added this as a disclaimer. I can remove this if needed.

Reply:
please remove that sentence